Content.Client/ContextMenu/UI/EntityMenuElement.cs

```diff
@@ -28,7 +28,7 @@ public partial class EntityMenuElement : ContextMenuElement
         public readonly Label CountLabel;
         public readonly SpriteView EntityIcon = new() { OverrideDirection = Direction.South};
 
-        public EntityMenuElement(EntityUid entity = default)
+        public EntityMenuElement(EntityUid? entity = null)
```

Comment:
there is a default check in this method that should become a null check

Reply:
Oh yup. Also updated Dispose() to set to null.